dev-libs/libxslt/libxslt-1.1.34.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Let's not do this in this commit.

Reply:
Isn't EAPI=7 needed for BDEPEND?

Reply:
Yes, but for that reason you wouldn't do this in BDEPEND, but EAPI-6 DEPEND